modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -112,27 +110,65 @@ function perflab_aea_enqueued_js_assets_test() {
 			'color' => 'blue',
 		),
 		'description' => sprintf(
-		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
-			'<p>' . esc_html__( 'The amount of %1$s enqueued %2$s (size: %3$s) is acceptable.', 'performance-lab' ) . '</p>',
-			$enqueued_scripts,
-			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
-			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+			'<p>%s</p>',
+			esc_html(
+				sprintf(
+				/* translators: 1: Number of enqueued styles. 2.Styles size. */
+					_n(
+						'The amount of %1$s enqueued script (size: %2$s) is acceptable.',
+						'The amount of %1$s enqueued scripts (size: %2$s) is acceptable.',
+						$enqueued_scripts,
+						'performance-lab'
+					),
+					$enqueued_scripts,
+					size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+				)
+			)
 		),
 		'actions'     => '',
 		'test'        => 'enqueued_js_assets',
 	);
 
-	if ( $enqueued_scripts > 30 || perflab_aea_get_total_size_bytes_enqueued_scripts() > 300000 ) {
+	/**
+	 * Filters number of enqueued scripts to trigger warning.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int $scripts_limit Scripts limit number. Default '30'.
+	 */
+	$scripts_limit = apply_filters( 'perflab_aea_enqueued_scripts_limit', 30 );
+
+	/**
+	 * Filters size of enqueued scripts to trigger warning.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int $scripts_size_limit Enqueued Scripts size (in bytes) limit. Default '300000'.
+	 */
+	$scripts_size_limit = apply_filters( 'perflab_aea_enqueued_scripts_byte_size_limit', 300000 );
+
+	if ( $enqueued_scripts > $scripts_limit || perflab_aea_get_total_size_bytes_enqueued_scripts() > $scripts_size_limit ) {
 		$result['status']         = 'recommended';
 		$result['badge']['color'] = 'orange';
-		$result['description']    = sprintf(
-		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
-			'<p>' . esc_html__( 'Your website enqueues %1$s %2$s (size: %3$s). Try to reduce the number or to concatenate them.', 'performance-lab' ) . '</p>',
-			$enqueued_scripts,
-			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
-			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+
+		$result['description'] = sprintf(
+			'<p>%s</p>',
+			esc_html(
+				sprintf(
+				/* translators: 1: Number of enqueued styles. 2.Styles size. */
```

Comment:
See above, can you indent this by one more tab so that it's aligned with the function call below?